src/components/vacation-requests-table/vacation-requests-table-columns.ts

```diff
@@ -0,0 +1,78 @@
+import { GridColDef } from "@mui/x-data-grid";
+import strings from "../../localization/strings";
+import { DateTime } from "luxon";
+import { useAtomValue } from "jotai";
+import { languageAtom } from "../../atoms/languageAtom";
+
+const VacationRequestsTableColumns = () => {
+  const language = useAtomValue(languageAtom);
+
+  /**
+   * Format date
+   *
+   * @param date datetime object
+   * @returns formatted date time
+   */
+  function formatDate(date: DateTime, dateWithTime?: boolean) {
+    if (!date) {
+      return "";
+    }
+    return date
+      .setLocale(language)
+      .toLocaleString(dateWithTime ? DateTime.DATETIME_SHORT : undefined);
+  }
```

Comment:
missing ;

Reply:
If you mean line 21, there shouldn't be one.

Reply:
Rome will automatically add ; every time I save a file, so no need to worry about that.